### ISSUE TITLE
'make' fails

### ISSUE THREAD

**Issue by meowsbits** [ethereumproject/go-ethereum#716]
```
ubp52 ~/go/src/github.com/ethereumproject/go-ethereum master=
$ make
mkdir -p ./bin && GO111MODULE=on go build -o bin/abigen ./cmd/abigen
go: finding github.com/hashicorp/golang-lru v0.5.1
go: finding github.com/syndtr/goleveldb v0.0.0-20171214120811-34011bf325bc
go: finding github.com/ethereumproject/go-ethereum/accounts/abi/bind v0.0.0-20190521151733-fe17e9e1e2ce
go: finding github.com/gizak/termui v2.3.0+incompatible
go: finding github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
go: finding github.com/maruel/panicparse v1.2.0
go: finding github.com/rs/cors v0.0.0-20170727213201-7af7a1e09ba3
go: finding golang.org/x/sys v0.0.0-20190419153524-e8e3143a4f4a
go: finding golang.org/x/net v0.0.0-20190419010253-1f3472d942ba
go: finding github.com/mailru/easyjson v0.0.0-20190403194419-1ea4449da983
go: finding github.com/ethereumproject/ethash v0.0.0-20190401191819-b3fdb17512de
go: finding gopkg.in/urfave/cli.v1 v1.17.0
go: finding gopkg.in/fatih/set.v0 v0.1.0
go: finding github.com/peterh/liner v1.1.0
go: finding golang.org/x/tools v0.0.0-20190520220859-26647e34d3c0
go: finding gopkg.in/karalabe/cookiejar.v2 v2.0.0-20150724131613-8dcd6a7f4951
go: finding github.com/denisbrodbeck/machineid v0.8.0
go: finding github.com/mattn/go-isatty v0.0.7
go: finding github.com/robertkrimen/otto v0.0.0-20180617131154-15f95af6e78d
go: finding github.com/mattn/go-colorable v0.1.1
go: finding github.com/ethereumproject/benchmark v0.0.0-20190401191651-0f5bf26f7cd8
go: finding golang.org/x/crypto v0.0.0-20190418165655-df01cb2cc480
go: finding github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
go: finding golang.org/x/sys v0.0.0-20190403152447-81d4e9dc473e
go: finding github.com/spf13/afero v1.2.2
go: finding github.com/mattn/go-colorable v0.1.2
go: finding golang.org/x/sys v0.0.0-20190222072716-a9d3bda3a223
go: github.com/ethereumproject/ethash@v0.0.0-20190401191819-b3fdb17512de: unknown revision b3fdb17512de
go: github.com/ethereumproject/benchmark@v0.0.0-20190401191651-0f5bf26f7cd8: unknown revision 0f5bf26f7cd8
go: github.com/ethereumproject/go-ethereum/accounts/abi/bind@v0.0.0-20190521151733-fe17e9e1e2ce: parsing go.mod: unexpected module path "github.com/eth-classic/go-ethereum/accounts/abi/bind"
go: error loading module requirements
Makefile:27: recipe for target 'cmd/abigen' failed
make: *** [cmd/abigen] Error 1
ubp52 ~/go/src/github.com/ethereumproject/go-ethereum master=
$ git rev-parse HEAD
498eb630a8e5e72d2f2e4f8cafeab3ad8ee19b0a
ubp52 ~/go/src/github.com/ethereumproject/go-ethereum master=
$ uname -a
Linux ubp52 4.17.4-041704-generic #201807031235 SMP Tue Jul 3 12:37:29 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
ubp52 ~/go/src/github.com/ethereumproject/go-ethereum master=
$ go version
go version go1.12.1 linux/amd64
ubp52 ~/go/src/github.com/ethereumproject/go-ethereum master=
$ go env
GOARCH="amd64"
GOBIN="/home/ia/go/bin"
GOCACHE="/home/ia/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/ia/go"
GOPROXY=""
GORACE=""
GOROOT="/home/ia/go1.12.1.linux-amd64/go"
GOTMPDIR=""
GOTOOLDIR="/home/ia/go1.12.1.linux-amd64/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build147925054=/tmp/go-build -gno-record-gcc-switches"
ubp52 ~/go/src/github.com/ethereumproject/go-ethereum master=
$
```

**Comment by meowsbits** [comment]
Looks like same or very similar error on CI - https://circleci.com/gh/ethereumproject/go-ethereum/4452?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link. Rel 498eb630a8e5e72d2f2e4f8cafeab3ad8ee19b0a

**Comment by soc1c** [comment]
Does it also happen building https://github.com/eth-classic/go-ethereum ?

**Comment by soc1c** [comment]
that's actually a replace issue

```
go: github.com/ethereumproject/ethash@v0.0.0-20190401191819-b3fdb17512de: unknown revision b3fdb17512de
go: github.com/ethereumproject/benchmark@v0.0.0-20190401191651-0f5bf26f7cd8: unknown revision 0f5bf26f7cd8
go: github.com/ethereumproject/go-ethereum/accounts/abi/bind@v0.0.0-20190521151733-fe17e9e1e2ce: parsing go.mod: unexpected module path "github.com/eth-classic/go-ethereum/accounts/abi/bind"
```

the revision is on eth-classic

**Comment by meowsbits** [comment]
seems to have been resolved for me with #715
